configs/KAKUTEF7/config.h

```diff
@@ -79,33 +81,29 @@
 #define USB_DETECT_PIN       PA8
 
 #define TIMER_PIN_MAPPING \
-    TIMER_PIN_MAP( 0, PE13, 1,  1) \
-    TIMER_PIN_MAP( 1, PB0 , 2,  0) \
-    TIMER_PIN_MAP( 2, PB1 , 2,  0) \
-    TIMER_PIN_MAP( 3, PE9 , 1,  2) \
-    TIMER_PIN_MAP( 4, PE11, 1,  1) \
-    TIMER_PIN_MAP( 5, PC9 , 2,  0) \
-    TIMER_PIN_MAP( 6, PA3 , 2,  0) \
-    TIMER_PIN_MAP( 7, PD12, 1,  0)
-
+    TIMER_PIN_MAP( 0, RX_PPM_PIN   , 1,  1) \
```

Comment:
```suggestion
    TIMER_PIN_MAP( 0, RX_PPM_PIN   , 1,  -1) \
```
RX_PPM should probably be removed as the timer conflicts with Motors 3 and 4 and would not be usable anyway.